assets/src/domain/index.ts

```diff
@@ -0,0 +1,14 @@
+import './shared/services/publicPath';
+
+const domain = window?.eeDomain;
+
+if (domain) {
+	import(
+		/* webpackExclude: /(shared)/ */
+		/* webpackChunkName: "[request]" */
+		`./${domain}/main.ts`
```

Comment:
`main.ts` is good, but I'm thinking we could make it 100% explicitly clear by using something like `entry-point.ts`

Reply:
Agreed

Reply:
Done